lib/schemas.js

```diff
@@ -19,7 +18,7 @@ function Schemas () {
 }
 
 Schemas.prototype.add = function (inputSchema) {
-  var schema = fastClone(inputSchema[kFluentSchema]
+  var schema = fastClone(inputSchema.isFluentSchema
```

Comment:
Can you please keep the Symbol check as well? In case somebody still use an old version

Reply:
Done.